setup.py

```diff
@@ -448,7 +454,7 @@ def _remove_extension(self, name: str) -> None:
     def get_macos_sdk_path(self) -> str | None:
         try:
             sdk_path = (
-                subprocess.check_output(["xcrun", "--show-sdk-path"])
+                subprocess.check_output(["xcrun", "--show-sdk-path", "--sdk", "macosx"])
```

Comment:
What was the reason for this change?

Reply:
Explicitness. Xcode can manage multiple SDKs (iOS being an obvious one); adding the explicit `--sdk macosx` SDK ensures that you're *definitely* getting the macOS one. That will be the default on *almost* every install of Xcode, but if you have a stray `SDKROOT` environment variable from some other activity, it could be inadvertently pointing at an iOS, tvOS, visionOS or MacCatalyst SDK.